### PR TITLE
Update slack documentation

### PR DESCRIPTION
**Pull request by fydai**

### CODE REVIEW

File: ocfweb/docs/docs/contact/slack.md [hunk 1]
```diff
@@ -6,18 +6,50 @@ We also offer IRC as a chat service. If that is what you were looking for head o
...
-an account with your <ocfusername>@ocf.berkeley.edu email address. This will
-*only* work with an OCF email address. Note that your OCF email by default will
-forward to your CalNet email address (e.g. oski@berkeley.edu). Using the
-confirmation link in your email, you should now be able to set a display name
-and password and join the OCF workspace. We recommend setting your display name
-to your OCF username for consistency.
+1. Create an OCF account [here][join].
+
+2. Go to the OCF Slack workspace [here][slack] and click the "If you have an
+@ocf.berkeley.edu email address, you can create an account." button.
```
Comment (jvperrin) [inline ocfweb/docs/docs/contact/slack.md]: nit: Indent to match the numbered list, like the other ones in this list

File: ocfweb/docs/docs/contact/slack.md [hunk 2]
```diff
@@ -6,18 +6,50 @@ We also offer IRC as a chat service. If that is what you were looking for head o
...
+
+3. Enter your OCF username in the box, this should send an email your Berkeley
+   email. If you don't recieve an email, please contact a staff member for
+   assistance.
+
+4. Click the link in your Berkeley email and follow the instructions to complete
+   setup. We recommend setting your display name to your OCF username for
```
Comment (jvperrin) [inline ocfweb/docs/docs/contact/slack.md]: I'd go with "strongly recommend" honestly, but very much a nitpick, so feel free to disregard

File: ocfweb/docs/docs/contact/slack.md [hunk 3]
```diff
@@ -6,18 +6,50 @@ We also offer IRC as a chat service. If that is what you were looking for head o
...
+
+*#announcements*: Low volume announcements channel
+
+*#administrivia*: Administrative discussion
+
+*#henlo*: Social chat for current staff
+
+*#ocf*: General alumni hangout channel and non-OCF tech discussion
+
+*#rebuild*: Technical discussion
```
Comment (jvperrin) [inline ocfweb/docs/docs/contact/slack.md]: I'd also mention `#rebuild-spam` in here as it's an actually useful channel for seeing changes/PRs come through, though also mostly spam

**Comment by jvperrin** [comment]
lol race conditions, oh well

File: ocfweb/docs/docs/contact/slack.md [hunk 4]
```diff
@@ -6,18 +6,50 @@ We also offer IRC as a chat service. If that is what you were looking for head o
...
-forward to your CalNet email address (e.g. oski@berkeley.edu). Using the
-confirmation link in your email, you should now be able to set a display name
-and password and join the OCF workspace. We recommend setting your display name
-to your OCF username for consistency.
+1. Create an OCF account [here][join].
+
+2. Go to the OCF Slack workspace [here][slack] and click the "If you have an
+@ocf.berkeley.edu email address, you can create an account." button.
+
+3. Enter your OCF username in the box, this should send an email your Berkeley
```
Comment (dkess) [inline ocfweb/docs/docs/contact/slack.md]: nit: fix typo

**Comment by cg505** [comment]
We should merge the channel information with the IRC page. Also, don't you have to put username@ocf.berkeley.edu, not _just_ the username in the box during sign-up?

**Comment by ethanhs** [comment]
>don't you have to put username@ocf.berkeley.edu, not just the username in the box during sign-up

This is quite possible, if so we should fix this.

>We should merge the channel information with the IRC page

I think we can create an issue for that

**Comment by cg505** [comment]
nvm, its ok. it looks like this 
image

However, we should mention that when logging in you need to do <username>@ocf.berkeley.edu

**Comment by cg505** [comment]
>I think we can create an issue for that

done

**Pull request merged**